cosmo_tester/framework/test_hosts.py

```diff
@@ -247,6 +243,8 @@ def create(
         os.makedirs(self._tmpdir)
         self._openstack = util.create_openstack_client()
         self.additional_install_config = {}
+        if upload_plugins is None:
```

Comment:
is not?

Reply:
if self.upload_plugins is None:
(updated)